CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 This changelog track changes to the qollage project starting at version v0.1.0
 
+## 0.3.1
+
+* Use ureq's `native-certs` feature.
```

Comment:
Can you explain a little bit more what this is and what this is needed for?

Reply:
It's supposed to make the default TLS implementation use the OS’ trust store. The goal is to fix an issue where ureq fails to validate the certificate of certain files.